Library/Homebrew/tab.rb

```diff
@@ -39,11 +39,11 @@ def self.create(formula, compiler, stdlib)
       "runtime_dependencies"    => Tab.runtime_deps_hash(formula, runtime_deps),
       "arch"                    => Hardware::CPU.arch,
       "source"                  => {
-        "path"         => formula.specified_path.to_s,
-        "tap"          => formula.tap&.name,
-        "tap_git_head" => formula.tap&.git_head,
-        "spec"         => formula.active_spec_sym.to_s,
-        "versions"     => {
+        "path"             => formula.specified_path.to_s,
+        "tap"              => formula.tap&.name,
+        "tap_git_revision" => formula.tap&.git_head,
```

Comment:
Don't think we can rename this here without breaking existing tabs?

Reply:
That was my concern, but are we using this anywhere? Based on my (admittedly naïve) search, the only places with references to the `tap_git_head` key were the ones I've modified in this PR. I'd be happy to revert if this is a breaking change we'd want to avoid.

Reply:
This is effectively a public API given we're writing out a JSON file so: I'd rather not change it ❤️